resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicExecutorFeatureStep.scala

```diff
@@ -276,11 +276,6 @@ private[spark] class BasicExecutorFeatureStep(
       .editOrNewMetadata()
         .withName(name)
         .addToLabels(kubernetesConf.labels.asJava)
-        .addToLabels(SPARK_RESOURCE_PROFILE_ID_LABEL, resourceProfile.id.toString)
-        .addToLabels(
-          SPARK_APP_NAME_LABEL,
-          KubernetesConf.getAppNameLabel(kubernetesConf.appName)
-        )
```

Comment:
FYI, these labels are ready included in KubernetesExecutorConf.labels:
https://github.com/apache/spark/blob/068d53bd5d89c96bf0cdb05d3ec7f2f023cf3875/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesConf.scala#L158-L165